frontend/src/redux/sagas/restaurants/index.js

```diff
@@ -0,0 +1,19 @@
+import { call, put, takeEvery } from 'redux-saga/effects'
+import axios from 'axios';
+
+import { types, restaurantsActions } from '../../reducers/restaurants';
+
+export function* callGetRestaurants() {
+    try {
+        const API_PATH = 'http://localhost:3000/restaurants';
+        const { data } = yield axios.get(API_PATH);
+        console.log(data);
```

Comment:
I think general practice would be to keep the log statements out of code that is released to production.

Reply:
Forgot to remove this, I'll fix it.

Reply:
Fixed in [9b99bcd](https://github.com/NickPriv/FryRank/pull/17/commits/9b99bcd1c8795461a981ef323ec657bb2d09b4a1).

Reply:
I don't see it removed in the diff for that commit

Reply:
Must have forgot it, I'll fix tomorrow.

Reply:
Fixed in [e8f1695](https://github.com/NickPriv/FryRank/pull/17/commits/e8f1695f2a26e47e72060b6f2caab2542be76086)